src/APIs/Axios.ts

```diff
@@ -1,31 +1,25 @@
 import Axios, { type CreateAxiosDefaults, AxiosError } from 'axios';
 
-const getAxiosInstance = () => {
-  const axiosConfig: CreateAxiosDefaults = {
-    baseURL: '',
-    headers: {
-      'Content-Type': 'application/json',
-      'Access-Control-Allow-Origin': '*', // CORS 문제 해결
-    },
-    withCredentials: true,
-    timeout: 2500,
-  };
-
-  const axios = Axios.create(axiosConfig);
-
-  axios.interceptors.request.use(
-    // 요청이 성공했을 때 실행될 함수
-    (config) => {
-      return config;
-    },
-    // 요청이 실패했을 때 실행될 함수
-    (error: AxiosError) => {
-      console.log('axios request error', error);
-      return Promise.reject(error);
-    },
-  );
-
-  return axios;
+const axiosConfig: CreateAxiosDefaults = {
+  baseURL: '',
+  headers: {
+    'Content-Type': 'application/json',
+    'Access-Control-Allow-Origin': '*', // CORS 문제 해결
+  },
+  withCredentials: true,
+  timeout: 2500,
 };
 
-export const axios = getAxiosInstance;
+export const axios = Axios.create(axiosConfig);
+
+axios.interceptors.request.use(
+  // 요청이 성공했을 때 실행될 함수
+  (config) => {
+    return config;
+  },
+  // 요청이 실패했을 때 실행될 함수
+  (error: AxiosError) => {
+    console.log('axios request error', error);
+    return Promise.reject(error);
+  },
+);
```

Comment:
export 다음에 코드를 작성해도 JS Module 특성상 평가에는 문제가 없으나 보통 인스턴스를 생성해줄 땐 설정을 마친 뒤 내보내는 순서가 유지보수 및 가독성에 이점을 가질 수 있는 방법인 것 같아요! 다음과 같이요.

```suggestion
const axios = Axios.create(axiosConfig);

axios.interceptors.request.use(
  // 요청이 성공했을 때 실행될 함수
  (config) => {
    return config;
  },
  // 요청이 실패했을 때 실행될 함수
  (error: AxiosError) => {
    console.log('axios request error', error);
    return Promise.reject(error);
  },
);

export { axios }
```